.kitchen.dokken.yml

```diff
@@ -27,24 +31,15 @@ platforms:
       intermediate_instructions:
         - RUN /usr/bin/apt-get update
 
-  - name: centos-6
-    driver:
-      image: dokken/centos-6
-      platform: rhel
-      pid_one_command: /sbin/init
-
   - name: centos-7
     driver:
       image: dokken/centos-7
-      platform: rhel
       pid_one_command: /usr/lib/systemd/systemd
 
-  - name: ubuntu-16.04
+  - name: fedora-28
     driver:
-      image: dokken/ubuntu-16.04
-      pid_one_command: /sbin/init
-      intermediate_instructions:
-        - RUN /usr/bin/apt-get update
+      image: dokken/fedora-28
+      pid_one_command: /usr/lib/systemd/systemd
```

Comment:
Fedora is a horrible moving target that never really gets fixes, and increases "bugs" in the cookbooks. 

We should put this in our official docs that we don't support Fedora, but it *probably* works